scripts/new_project.kts

```diff
@@ -54,6 +58,27 @@ object NewProject {
         }
     }
 
+    private suspend fun cleanOldBuildFiles() {
+        withContext(coroutineContext) {
+            async {
+                // Clean old build
+                "sh \"../$appNameWithoutSpace\"/gradlew -p \"../$appNameWithoutSpace\" clean".exec()
```

Comment:
Hmmm, I'm not a big fan of this approach, as it requires to escape `"` with `\`

Have you tried [my approach](https://github.com/nimblehq/android-templates/compare/kscript-poc?expand=1#diff-bc4c132346cb8367e875b5d0cf2ff92b86d04b2b7ad9bbef0dd1674c9a649ceaR116-R122), it requires a lot less code.

If yes, any difference? 🙏

Reply:
I've tried running with `./` before but it says `permission denied`. Also I could not run the bash using the proposed approach. Please let me know if anyone can. 🙏

Reply:
Hmmm, I see. I'll try again soon, maybe I overlooked something when I tried before.  🚀

Reply:
@kaungkhantsoe I could be wrong, but this is working for me. Please try it out 🤩  

```
private fun cleanOldBuildFiles() {
    executeCommand("./../$appNameWithoutSpace/gradlew -p ../$appNameWithoutSpace clean")
    executeCommand("./../$appNameWithoutSpace/gradlew -p ../$appNameWithoutSpace/buildSrc clean")
     ...
{

private fun executeCommand(command: String) {
    val process = Runtime.getRuntime().exec(command)
    InputStreamReader(process.inputStream).forEachLine {
        println(it)
    }
}
```

Reply:
@Tuubz I could run with it. I am not sure why I couldn't run it before 🧐. I've updated the code using your approach.